.github/workflows/lte-integ-test-bazel.yml

```diff
@@ -78,7 +78,9 @@ jobs:
         run: |
           cd lte/gateway
           vagrant ssh -c 'cd ~/magma; bazel/scripts/remote_cache_bazelrc_setup.sh "${{ env.CACHE_KEY }}" "${{ env.REMOTE_DOWNLOAD_OPTIMIZATION }}" "${{ secrets.BAZEL_REMOTE_PASSWORD }}";' magma
-          vagrant ssh -c 'cd ~/magma; bazel build --profile=bazel_profile_lte_integ_tests `bazel query "kind(.*_binary, //orc8r/... union //lte/... union //feg/...)"`;' magma
+          REGEX='"service|util_script"'
```

Comment:
If I locally run:
```
export REGEX='"service|util_script"'
export QUERY="'attr(tags, "${REGEX}", kind(.*_binary, //orc8r/... union //lte/... union //feg/... except //lte/gateway/c/core:mme_oai))'"
bazel query '"${QUERY}"'
```
I get `ERROR: no such target '//:${QUERY}': target '${QUERY}' not declared in package '' defined by /home/vagrant/magma/BUILD.bazel`. Have you tried to run this in CI?

Reply:
+ if I run the lines exactly as is from outside the VM I get `ERROR: /home/vagrant/magma/lte/gateway/release/BUILD.bazel:122:1: syntax error at '//': expected expression`

Reply:
Sorry about that - I think there is an error with my set up. I will try again.

Reply:
This works locally for me and it is also working in this fork run: https://github.com/vktng/magma/actions/runs/3275484631/jobs/5400996012

Reply:
I get this weird error:
```
ERROR: An error occurred during the fetch of repository 'bazel_gazelle_go_repository_tools':
   Traceback (most recent call last):
	File "/tmp/bazel/external/bazel_gazelle/internal/go_repository_tools.bzl", line 103, column 13, in _go_repository_tools_impl
		fail("failed to build tools: " + result.stderr)
Error in fail: failed to build tools: # github.com/bazelbuild/bazel-gazelle/language/proto
unexpected fault address 0xabcc
fatal error: fault
[signal SIGSEGV: segmentation violation code=0x1 addr=0xabcc pc=0x7512b1]
```
but my ping on github.com is normal